Drakhtar/EventListeners/StoreListener.cpp

```diff
@@ -76,15 +97,30 @@ StoreListener::~StoreListener() {}
 void StoreListener::reset() {
   storeUnit.amount_ = 0;
   storeUnit.amountText->setText(std::to_string(storeUnit.amount_));
+  storeUnit.amountText->setColor({255, 255, 255, 0});
+}
+
+std::string StoreListener::infoFillText() {
+  std::string info =
+      "-After each fight, the surviving units will be conserved, buying will "
+      "increase the number you currently have.\n";
+      "-The total amount you buy will be added to one single unit, giving it "
+      "more attack power and hp. \n";
+  info += "E.g: having 5 archers will multiply archers stats by 5 \n";
+  info +=
+      "-The higher the amount of one unit is, the higher the cost of the next "
+      "one will be. \n";
+  info += "-You can buy up to a certain number for each unit.";
+
+  return info;
```

Comment:
This can be done in another way:

```cpp
return "- After each fight, the surviving units will be conserved, buying will "
    "increase the number you currently have.\n"
    "- The total amount you buy will be added to one single unit, giving it "
    "more attack power and health points.\n"
    "E.g: having 5 archers will multiply archers stats by 5.\n"
    "- The higher the amount of one unit is, the higher the cost of the next "
    "one will be.\n"
    "- You can buy up to a certain number for each unit.";
```

I'm not really sure, I was not aware string concatenation without the `+` even existed in C++, but if this fails, just add a `+` at the end of each line.

Reply:
Done